cmd/launcher/options.go

```diff
@@ -53,6 +53,7 @@ func parseOptions(args []string) (*launcher.Options, error) {
 		flMirrorURL          = flagset.String("mirror_url", autoupdate.DefaultMirror, "The mirror server for autoupdates (default: https://dl.kolide.co)")
 		flAutoupdateInterval = flagset.Duration("autoupdate_interval", 1*time.Hour, "The interval to check for updates (default: once every hour)")
 		flUpdateChannel      = flagset.String("update_channel", "stable", "The channel to pull updates from (options: stable, beta, nightly)")
+		flGUNPrefix          = flagset.String("gun_prefix", autoupdate.DefaultGUNPrefix, "The prefix for the GUN that contains the collections (default: kolide/)")
```

Comment:
I would be inclined to call this `notary_prefix`. It's notary specific, and I find the `GUN` terminology somewhat distasteful.